flink-contrib/docker-flink/Dockerfile

```diff
@@ -36,22 +31,24 @@ ENV PATH $PATH:$FLINK_HOME/bin
 EXPOSE 8081
 EXPOSE 6123
 
+# flink-dist can point to a directory or a tarball on the local system
+ARG flink_dist=NOT_SET
+
 # Install build dependencies and flink
+ADD $flink_dist $FLINK_INSTALL_PATH
 RUN set -x && \
-  mkdir -p $FLINK_INSTALL_PATH && \
-  apk --update add --virtual build-dependencies curl && \
-  curl -s $(curl -s https://www.apache.org/dyn/closer.cgi\?preferred\=true)flink/flink-${FLINK_VERSION}/flink-${FLINK_VERSION}-bin-hadoop${HADOOP_VERSION}-scala_${SCALA_VERSION}.tgz | \
```

Comment:
Pefect, makes sense, but we should probably keep updating the existing image for some time, because this is the way most people are aware that it exists, or add a README that it was moved and maintained in the new repo.
Are you going to be the official maintainer (it is ok with me if you want to take the lead there, I just want to learn the whole process of creating an official image), but I can also help if you need with maintenance.